kratos/tests/cpp_tests/containers/test_pointer_vector_set.cpp

```diff
@@ -0,0 +1,40 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ruben Zorrilla
+//
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/element.h"
+#include "containers/model.h"
+#include "containers/pointer_vector_set.h"
+#include "testing/testing.h"
+
+namespace Kratos {
+namespace Testing {
+
+KRATOS_TEST_CASE_IN_SUITE(PointerVectorSetCBeginAndCEnd, KratosCoreFastSuite)
+{
+    PointerVectorSet<const Element> test_container;
+    auto p_element_1 = Kratos::make_intrusive<Element>(1);
+    auto p_element_2 = Kratos::make_intrusive<Element>(2);
+    test_container.push_back(p_element_1);
+    test_container.push_back(p_element_2);
```

Comment:
```suggestion
    auto p_element_1 = Kratos::make_intrusive<Element>(1);
    auto p_element_2 = Kratos::make_intrusive<Element>(2);
    auto p_element_2 = Kratos::make_intrusive<Element>(3);
    test_container.push_back(p_element_1);
    test_container.push_back(p_element_2);
    test_container.push_back(p_element_3);
```

Reply:
Why adding another one?

Reply:
I thought this makes the test more "real"
Having only two elements in there and then asking for begin and end doesn't seem very representative

Reply:
OK. I can add them.

Reply:
Done.